tests/src/Integration/Action/EntityCreateTest.php

```diff
@@ -60,31 +60,31 @@ public function setUp() {
     $bundle_field_definition->getItemDefinition()
       ->willReturn($item_definition->reveal());
     $bundle_field_definition->getCardinality()->willReturn(1)
-      ->shouldBeCalledTimes(1);
+      ->shouldBeCalled();
     $bundle_field_definition->getType()->willReturn('string');
     $bundle_field_definition->getLabel()->willReturn('Bundle')
-      ->shouldBeCalledTimes(1);
+      ->shouldBeCalled();
     $bundle_field_definition->getDescription()
       ->willReturn('Bundle mock description')
-      ->shouldBeCalledTimes(1);
+      ->shouldBeCalled();
 
     $bundle_field_definition_required->getItemDefinition()
```

Comment:
Are all these mocks really needed? Looks like left-overs from EntityCreateTest for me?